modules/osdk-updating-v1101-to-v1160.adoc

```diff
@@ -0,0 +1,189 @@
+----
+
+. Make the following changes to the `config/manager/manager.yaml` file:
+
+.. Increase the CPU and memory resource limits:
++
+[source,diff]
+----
+resources:
+  limits:
+-     cpu: 100m
+-     memory: 30Mi
++     cpu: 200m
++     memory: 100Mi
+----
+
+.. Add an annotation to specify the default container manager:
++
+[source,yaml]
+----
+...
+template:
+  metadata:
+    annotations:
+      kubectl.kubernetes.io/default-container: manager
+...
+----
+
+. Add `PHONY` targets to all of the targets in your `Makefile` file. [@osdk-dev: does this need to be expanded or have an example?]
+
+. For Go-based Operator projects, make the following changes:
+
+.. Install the `setup-envtest` binary. [@osdk-dev: Does this need to be expanded?]
+
+.. Change your `go.mod` file to update the [@osdk-dev: Are these Kubernetes or Operator SDK dependencies?] dependencies:
++
+[source,golang]
+----
+k8s.io/api v0.22.1
+k8s.io/apimachinery v0.22.1
+k8s.io/client-go v0.22.1
+sigs.k8s.io/controller-runtime v0.10.0
+----
+
+.. Run the `go mod tidy` command to download the dependencies:
++
+[source,terminal]
+----
+$ go mod tidy
+----
+
+.. Make the following changes to your `Makefile` file:
++
+[source,diff]
+----
+...
+- ENVTEST_K8S_VERSION = 1.21`
++ ENVTEST_K8S_VERSION = 1.22`
+
+  test: manifests generate fmt vet envtest ## Run tests.
+-   go test ./... -coverprofile cover.out
++   KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
+...
+
+- $(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
++ $(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+...
+
+# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
+- CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
+...
+- admissionReviewVersions={v1,v1beta1}
++ admissionReviewVersions=v1
+...
+
++ ifndef ignore-not-found
++   ignore-not-found = false
++ endif
+
+##@ Deployment
+...
+- sh kubectl delete -f -
++ sh kubectl delete --ignore-not-found=$(ignore-not-found) -f -
+----
+
+.. Run the `make manifest` command to generate your manifests with the updated version of Kubernetes:
++
+[source,terminal]
+----
+$ make manifest
+----
++
+[IMPORTANT]
+====
+The Operator SDK rolls back to earlier versions of Kubernetes to ensure compatibility with earlier cluster versions. However, many deprecated `v1beta1` APIs were removed in Kubernetes 1.22, including `sigs.k8s.io/controller-runtime v0.10.0` and `controller-gen v0.7`. This is a breaking change if you need to scaffold `v1beta1` APIs for custom resource definitions (CRDs) or webhooks to publish your project into older cluster versions.
+====
+
+. For Ansible-based Operator projects, make the following changes:
++
+.. Change your `requirements.yml` file to include the following:
+
+... Add the `kubernetes.core` collection:
```

Comment:
And remove the community.kubernetes collection

Reply:
Fixed!